package.json

```diff
@@ -5,15 +5,15 @@
         "type": "git",
         "url": "https://github.com/PropelAuth/react"
     },
-    "version": "2.0.20",
+    "version": "2.0.21",
```

Comment:
This looks differing from package-lock.json -- they should be the same version `2.0.21`